src/bootstrap-table.css

```diff
@@ -196,7 +196,7 @@
     line-height: 1.428571429;
 }
 
-.fixed-table-toolbar .bars,
+.fixed-table-toolbar .bars-bs-table,
```

Comment:
I think here is better to use `.bs-bars`, we use like `.bs-`... for example `.bs-checkbox` 
Do you think so? @djhvscf